tests/conftest.py

```diff
@@ -323,6 +326,31 @@ def ptfhost(ansible_adhoc, tbinfo, duthost):
         ptf_host = duthost.host.options["inventory_manager"].get_host(duthost.hostname).get_vars()["ptf_host"]
         return PTFHost(ansible_adhoc, ptf_host)
 
+@pytest.fixture(scope="module")
+def k8shosts(ansible_adhoc, request, creds):
+    """
+    Shortcut fixture for getting Kubernetes hosts
+    """
+    master_id = request.config.getoption("--kube_master") 
+    master_id_elements = master_id.split('_')
+    assert len(master_id_elements) == 2, "Invalid master identifier format. Please use format {servernumber}_{mastersetnumber}"
+    master_server_id = master_id_elements[0]
+    master_set_id = master_id_elements[1]
+    k8s_ansible_group = "k8s_vms{}_{}".format(master_set_id, master_server_id)
```

Comment:
IMO, the design of `--kube_master` argument is a little bit confusing. Why not we just pass value like `k8s_vms1_21` to argument `--kube_master`?

Reply:
Thanks for the suggestion, I changed it to `k8s_vms1_21` format